frontend/src/components/material/MaterialTable.vue

```diff
@@ -27,6 +27,14 @@
       </tr>
     </template>
 
+    <template #[`item.done`]="{ item }">
+      <api-checkbox
+        :disabled="layoutMode || doneChecklistDisabled || disabled"
```

Comment:
Nice that you considered layoutMode, but in the activity detail and category detail view, doneChecklistDisabled is always set, isn't it?